src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php

```diff
@@ -98,7 +98,7 @@ class RaSecondFactor implements JsonSerializable
     public $email;
 
     /**
-     * @ORM\Column(type="stepup_document_number")
+     * @ORM\Column(type="stepup_document_number", nullable=true)
```

Comment:
Would have been nice to include in the commit message why this change was made 0e3949e23b2ebdb30583f30342a44854eda4ac28